### ISSUE TITLE
AST and CSG output is written to the wrong directory

### ISSUE THREAD

**Issue by kintel** [openscad/openscad#4632]
`openscad /path/to/file.scad -o out.csg` writes "out.csg" into "/path/to/file/" instead of to the current directory.
Same for .ast files.

Possibly related to the discussion here: https://github.com/openscad/openscad/issues/3879#issuecomment-918571018

To reproduce in tests, re-enable these two tests (https://github.com/openscad/openscad/blob/master/tests/CMakeLists.txt#L1355): 
* relative-output_csg_check
* relative-output_ast_check

**Comment by burunduk3** [comment]
I don't think it was fixed in any way — and the current behavior is very annoying when **openscad** is used in any kind of scripting environment (auto-generate models, build setups etc).

**Comment by kintel** [comment]
Agreed, it's a pretty clear bug

**Comment by jordanbrown0** [comment]
Duplicate of #4401.
